NetworkMonitor/Classes/FNMNetworkMonitor.swift

```diff
@@ -286,6 +294,11 @@ extension FNMNetworkMonitor {
 
 extension FNMNetworkMonitor: FNMNetworkMonitorURLProtocolDataSource {
 
+    func shouldIgnoreRequest(with url: URL) -> Bool {
+
+        self.ignoredDomains.contains(where: { url.absoluteString.range(of: $0) != nil } )
```

Comment:
Try to use `contains` instead of `range of`

Reply:
Done